apps/src/lib/kits/maker/boards/microBit/MBFirmataWrapper.js

```diff
@@ -12,7 +13,7 @@ export default class MicrobitFirmataWrapper extends MBFirmataClient {
     return Promise.resolve()
       .then(() => this.connect())
       .then(() => {
-        return this.setAnalogSamplingInterval(50);
+        return this.setAnalogSamplingInterval(SAMPLE_RATE);
```

Comment:
Aha, as commented already, sampling rate is not the same as sampling interval :)